extension/sidebar-pane.js

```diff
@@ -38,13 +43,28 @@ async function _updateSubtree(selectedNode) {
   progressEl.textContent = "Getting web compatibility issues";
   issueListEl.innerHTML = "";
   _recursiveNodesIssuesRendering(0, nodesInSubtree, false, issueListEl, []).then(() => {
+    if (!_isCurrentProcess(issueListEl)) {
+      return;
+    }
+
     subtreeEl.classList.remove("processing");
     if (!issueListEl.querySelector("li")) {
       _renderNoIssue(issueListEl);
     }
   });
 }
 
+function _getNewULElement(selector) {
```

Comment:
I think it would be better to simply call this `_getIssuesListRootElement`, but add a jsdoc comment before it that explains that every time it's called, it will actually insert a new element and replace the old one with it.